### DISCUSSION TITLE
IES vs Randomized JCO

### DISCUSSION THREAD

**Discussion by laat0003** [gmdsi/GMDSI_notebooks#106] [Q&A] [answered]
Howdy,

I'm an IES fanboi but my workplace colleagues favor the randomized JCO method in PEST_HP. Should I attempt to convert those heathens, or are there tangible benefits in using one method as opposed to the other? 
And ... just for fun ... would I get a different answer if I asked John D. the same question?

Cheers,
Tariq

**Comment by jtwhite79** [comment, marked as answer]
[https://ngwa.onlinelibrary.wiley.com/doi/10.1111/gwat.13106](https://ngwa.onlinelibrary.wiley.com/doi/10.1111/gwat.13106) 

That paper compares the two.  

In general i think IES can scale to larger numbers of parameters before running into memory issues (bc it solves in ensemble space).   Additionally ies yields a posterior ensemble, while randomized jco supports more advanced regularization techniques so it ought to be a candidate if you need a single minimum error variance solution.  But otherwise i think they function similarly.  And at the end of the day,  all of this is approximate and based on imperfect models so I'm not one to judge preferences! 

@doherty1234 agree?

**Comment by laat0003** [comment]
Great! Thanks for pointing out the paper. I was hoping that there was one. I appreciate the pragmatic response.
